### PR TITLE
Fix REST API docs

### PR DESCRIPTION
**Pull request by ruthryi**
Dev portal : Removed `x-example`
Publisher : Updated `base_path`

### CODE REVIEW

**Review by msm1992 — APPROVED** [review]

File: en/docs/reference/product-apis/publisher-apis/publisher-v2/publisher-v2.md [hunk 1]
```diff
@@ -2,5 +2,5 @@
 template: templates/redoc.html
 ---
 
-<redoc spec-url='{{base_path}}/reference/product-apis/publisher-apis/publisher-v2/publisher-v2.yaml'></redoc>
+<redoc spec-url='https://apim.docs.wso2.com/en/4.0.0-dev/reference/product-apis/publisher-apis/publisher-v2/publisher-v2.yaml'></redoc>
```
Comment (shaniR) [inline en/docs/reference/product-apis/publisher-apis/publisher-v2/publisher-v2.md:5]: why is this necessary?

Comment (ruthryi) [reply en/docs/reference/product-apis/publisher-apis/publisher-v2/publisher-v2.md:5]: Because, we have used `https://apim.docs.wso2.com/en/next` in mkdocs.yml. So I'm trying to figure out whether it is causing the issue.

**Comment by Mariangela** [comment]
@ruthryi , looks like your local pack is not updated. coz currently in mkdocs.yml base_path points to https://apim.docs.wso2.com/en/4.0.0-dev
So ideally we should have {{base_path}}

**Comment by Mariangela** [comment]
@msm1992 , are you sure that we need to remove the "x-example" section? I think this is a mistake.
